package.json

```diff
@@ -18,13 +18,18 @@
   "module": "src/index.ts",
   "scripts": {
     "pretty": "yarn prettier '**/*.*'",
-    "lint": "yarn pretty --check && yarn eslint .",
-    "format": "yarn build && yarn pretty --write && yarn eslint --fix .",
+    "lint": "yarn build && yarn link-plugin && yarn pretty --check && yarn eslint .",
+    "format": "yarn build && yarn link-plugin && yarn pretty --write && yarn eslint --fix .",
     "build": "tsc",
-    "test": "yarn build && jest"
+    "test": "yarn build && jest",
+    "link-plugin": "yarn link && yarn link eslint-plugin-jest-formatting"
+  },
+  "peerDependencies": {
+    "eslint": "5.x"
```

Comment:
Even though we no longer import from `eslint` we should still keep it as a peer dependency per their docs. Removing the import might allow to actually supported older versions of eslint... I think i remember reading that the plugin stuff started around eslint 3 or 4... maybe.

I'll have to look into it.

Reply:
Awesome, thanks

Reply:
Astonishingly... they say `>=0.8.0`, so that's what I used. https://eslint.org/docs/developer-guide/working-with-plugins#peer-dependency